public/javascripts/global.js

```diff
@@ -188,6 +188,7 @@ function fetchQuiz(event) {
       type: 'get',
       dataType: 'json',
         success: function(response){
+          console.log(response);
           window.location.href = 'quiz?id='+response[0]["_id"]+'&team_id='+response[0]["team_id"]+'&league='+response[0]["league"];
```

Comment:
can we try updating this query so that it just returns the single object rather than an array so we don't have to do this ugly 0 index thing. Maybe we can get findOne to work?